src/components/CustomDevMenu/types.ts

```diff
@@ -0,0 +1,3 @@
+type CustomDevMenuElement = null | void;
```

Comment:
Change the type so that it is a function, and not the return type.

```ts
type X = {
    (): null | void;
    displayName: string;
};
```

You can use it like this:
```ts
const CustomDevMenu: CustomDevMenuElement = () => {};
```